src/main/webapp/resources/js/apis/pipelines/pipelines.js

```diff
@@ -3,7 +3,7 @@
  */
 import axios from "axios";
 
-const URL = `${window.TL.BASE_URL}pipelines/ajax`;
+const URL = `pipelines/ajax`;
```

Comment:
We've  been using  the format `ajax/NAME_OF_CONTROLLER` for all the other URLs. We should change this to `ajax/pipelines`

Reply:
Yah we really should standardize them.